views/email/email_partials/breach_alert.hbs

```diff
@@ -4,10 +4,10 @@
       </td>
       <td valign="top" style="vertical-align: top; padding-top: 0px; padding-right: 20px; padding-bottom: 10px;">
         <p class="body-copy" style="padding: 0px; margin: 0px; color: #333333; margin: 0px; font-weight: 500; font-size: 16px;">{{ breachAlert.Title }}</p>
-        <p class="body-copy" style="padding: 2px 0px 2px 0px; margin: 0px; color: #333333; font-weight: 400; font-size: 14px;">Breach date:<span class="body-copy" style="font-weight:300; padding-left: 6px; color: #333333">{{prettyDate breachAlert.BreachDate }}</span></p>
-        <p class="body-copy" style="padding: 2px 0px 2px 0px; margin: 0px; color: #333333; font-weight: 400; font-size: 14px;">Compromised accounts:<span class="body-copy" style="font-weight:300; padding-left: 6px; color: #333333">{{localeString breachAlert.PwnCount }}</span></p>
+        <p class="body-copy" style="padding: 2px 0px 2px 0px; margin: 0px; color: #333333; font-weight: 400; font-size: 14px;">{{fluentFormat supportedLocales "breach-date"}}:<span class="body-copy" style="font-weight:300; padding-left: 6px; color: #333333">{{prettyDate supportedLocales breachAlert.BreachDate }}</span></p>
+        <p class="body-copy" style="padding: 2px 0px 2px 0px; margin: 0px; color: #333333; font-weight: 400; font-size: 14px;">{{fluentFormat supportedLocales "compromised-accounts"}}:<span class="body-copy" style="font-weight:300; padding-left: 6px; color: #333333">{{localeString breachAlert.PwnCount }}</span></p>
```

Comment:
Filed https://github.com/mozilla/blurts-server/issues/547.

Reply:
Doesn't need to block this merge - I'd like to merge the .ftl files ASAP so we can start translating. #547 is part of l10n QA bug-fixing. 😄